perf/benchmark/runner/runner.py

```diff
@@ -132,7 +132,7 @@ def __init__(
         self.protocol_mode = protocol_mode
         self.ns = NAMESPACE
         # bucket resolution in seconds
-        self.r = "0.00005"
+        self.r = "0.1"
```

Comment:
ahem, no this is in seconds, so for 10ms you would want ~ 1ms resolution, ie 0.001

Reply:
oh, I see, let me update it, thanks!

Reply:
also test/look at the resulting json with different values to confirm it helps 
(the goal is that the buckets match the data fine enough to get correct p90 etc calculations, and be comparable with nighthawk's)